cypress/integration/api-user-creation.spec.js

```diff
@@ -0,0 +1,119 @@
+/// <reference types='Cypress' />
+
+import {endpoints} from '../support/endpoints.js';
```

Comment:
Consider using any linter that helps you format the code properly, like leaving spaces where space should go